jsnip/src/main/java/dev/marcelo/java/Caller.java

```diff
@@ -19,15 +19,15 @@ public int getSomeInt() {
   }
 
   public void setSomeInt(int someInt) {
-    this.someInt = Callee.someInt + someInt;
+    this.someInt = Callee.someInt + someInt + 1;
```

Comment:
[reviewpad-sig]: <>
[Linked to](/review/github.com/marcelosousa/java-snippets/pull/3#comment-ca1ff4de-ee2a-456c-8653-84e8393b00b0)\n